com.unity.netcode.gameobjects/Runtime/Simulator/INetworkEventsApi.cs

```diff
@@ -1,4 +1,15 @@
-﻿using System;
+﻿// NetSim Implementation compilation boilerplate
+// All references to UNITY_MP_TOOLS_NETSIM_ENABLED should be defined in the same way,
+// as any discrepancies are likely to result in build failures
+// ---------------------------------------------------------------------------------------------------------------------
+#if UNITY_EDITOR || ((DEVELOPMENT_BUILD && !UNITY_MP_TOOLS_NETSIM_DISABLED_IN_DEVELOP) || (!DEVELOPMENT_BUILD && UNITY_MP_TOOLS_NETSIM_ENABLED_IN_RELEASE))
```

Comment:
**Suggestion**: You can probably remove one parenthesis here without affecting the result:

```
#if UNITY_EDITOR || (DEVELOPMENT_BUILD && !UNITY_MP_TOOLS_NETSIM_DISABLED_IN_DEVELOP) || (!DEVELOPMENT_BUILD && UNITY_MP_TOOLS_NETSIM_ENABLED_IN_RELEASE)
```